packages/nomad/api.hcl

```diff
@@ -73,6 +73,9 @@ job "api" {
         REDIS_URL                     = "${redis_url}"
         # This is here just because it is required in some part of our code which is transitively imported
         TEMPLATE_BUCKET_NAME          = "skip"
+        ADMIN_TOKEN                   = var.admin_token
+        REDIS_URL                     = var.redis_url
```

Comment:
this duplicates line 72-73, which is either unnecessary or confusing 😅

Reply:
sorry, some mistake in merge conflict